### ISSUE TITLE
Consistent path to files

### ISSUE THREAD

**Issue by jansule** [o2r-project/o2r-meta#32]
It would be nice, if all paths to files in the metadata (i.e. `files` and `metadata.o2r`) would be consistent. Thus, it would be easier to work with them. Right now, in some cases the API endpoints are shown, in other cases absolute paths to folders on the server are shown.

I would suggest only using the API endpoints.

**Edit:** Here some examples:
Extract from `metadata.o2r.inputfiles`:
``` 
/tmp/o2r/compendium/lVAo0/data/Supplement/OhridComm.csv
```
Extract from `files`:
```
/api/v1/compendium/lVAo0/data/bag-info.txt
```

Extract from `metadata.o2r.file.filepath`:
```
lVAo0/data/Supplement/auxiliaryFunctions.R
```

**Comment by nuest** [comment]
My suggestion would be to always start with the compendium ID, so in the examples above `1VAo0`.

The root path `/tmp/o2r/...` is internal information that must not be publicly available.

**Comment by 7048730** [comment]
It feels to me like we changed this many times. So which of those are from extractor at all?
Only _inputfiles_ needs to be fixed, right?

**Comment by nuest** [comment]
yes, `inputfiles` (probably from extractor) should be changed.

@jansule is it no problem to create the full api path on the client side if we strip the `/api/v1/compendium/` from `files`?

**Comment by jansule** [comment]
@nuest stripping `/api/v1/compendium/` is **no problem**. To me, it is only important to have a consistent structure.

**Comment by 7048730** [comment]
as for the extractor, the path in `inputfiles` is constructed in the same way as the path from `files`
So this difference, I cannot fix by changing the extractor

```
/tmp/o2r/compendium/lVAo0/data/Supplement/OhridComm.csv`
```
```
lVAo0/data/Supplement/auxiliaryFunctions.R
```

See for yourselves:
https://github.com/o2r-project/o2r-meta/blob/master/extract/metaextract.py#L684
https://github.com/o2r-project/o2r-meta/blob/master/extract/metaextract.py#L688
https://github.com/o2r-project/o2r-meta/blob/master/extract/metaextract.py#L422-L424

**Comment by nuest** [comment]
I don't understand what you mean with "I cannot fix by changing the extractor". Can you not strip the leading path? So the extractor only creates path relative to the one that is given via `-i` ?

**Comment by 7048730** [comment]
the leading path is relative

**Comment by nuest** [comment]
I just uploaded the metatainer and also found this path:

```json
"viewfile": [
    "/tmp/o2r/compendium/F0BIZ/data/document.html"
]
```

**Comment by 7048730** [comment]
included with 7e1b08b95d998d06c1c4a108a6836f9cbaf560d7
use `-b` | `-basedir` as starting point directory for relative paths
